### ISSUE TITLE
import intake causes UnboundLocalError: local variable 'prefix' referenced before assignment

### ISSUE THREAD

**Issue by pydatawrangler** [ContinuumIO/intake#123]
I conda install intake according to the docs:  `conda install -c intake intake`, I get an "UnboundLocalError: local variable 'prefix' referenced before assignment".  Is there a fix for this?  Here is the full stack trace:

---------------------------------------------------------------------------
UnboundLocalError                         Traceback (most recent call last)
<ipython-input-2-924f1ec0a922> in <module>()
----> 1 import intake

\Anaconda3\lib\site-packages\intake\__init__.py in <module>()
     67 
     68 make_open_functions()
---> 69 cat = load_combo_catalog()
     70 
     71 

\Anaconda3\lib\site-packages\intake\catalog\default.py in load_combo_catalog()
     88     """Load a union of the user and global catalogs for convenience"""
     89     user_dir = user_data_dir()
---> 90     global_dir = global_data_dir()
     91 
     92     cat_dirs = []

\Anaconda3\lib\site-packages\intake\catalog\default.py in global_data_dir()
     78         prefix = conda_prefix()
     79 
---> 80     if prefix:
     81         # conda and virtualenv use Linux-style directory pattern
     82         return os.path.join(prefix, 'share', 'intake')

UnboundLocalError: local variable 'prefix' referenced before assignment

**Comment by martindurant** [comment]
Thank you @pydatawrangler for reporting.
This happens when software tries to be adaptable on many different installation types. Your help in testing this would be much appreciated.

**Comment by martindurant** [comment]
A probably fix is sitting in PR #124 .
To test that, you would clone this repo, change branch to the one with the fix, install the source, and test to see if you still have the problem.
```
git clone https://github.com/ContinuumIO/intake
cd intake
git checkout prefix_fix  # the name of the branch is at the top of the PR page
pip install -e . --no-deps
ipython
In [1]: import intake
```
Note that this will *overwrite* you existing install of intake, so you might want to do it in a fresh environment, or otherwise somewhere that you are confident you can reverse the action.

**Comment by pydatawrangler** [comment]
This worked.

However, I first got an "ImportError: No module named ruamel.yaml", and had
to `pip install ruamel.yaml`

Thanks,
